Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -0,0 +1,158 @@
+//swiftlint:disable todo
+
+import XCTest
+
+class UserFeedbackUITests: BaseUITest {
+    override var automaticallyLaunchAndTerminateApp: Bool { false }
+    
+    override func setUp() {
+        super.setUp()
+        app.launchArguments.append(contentsOf: [
+            "--io.sentry.iOS-Swift.auto-inject-user-feedback-widget",
+            "--io.sentry.iOS-Swift.user-feedback.all-defaults"
+        ])
+        launchApp()
+    }
+    
+    func testSubmitFullyFilledForm() throws {
+        let widgetButton: XCUIElement = app.staticTexts["Report a Bug"]
+        widgetButton.tap()
+        
+        let nameField: XCUIElement = app.textFields["Your Name"]
+        nameField.tap()
+        nameField.typeText("Andrew")
+        
+        let emailField: XCUIElement = app.textFields["your.email@example.org"]
+        emailField.tap()
+        emailField.typeText("andrew.mcknight@sentry.io")
+        
+        let messageTextView: XCUIElement = app.textViews["What's the bug? What did you expect?"]
+        messageTextView.tap()
+        messageTextView.typeText("UITest user feedback")
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        // displaying the form again ensures the widget button still works afterwards; also assert that the fields are in their default state to ensure the entered data is not persisted between displays
+        
+        widgetButton.tap()
+        
+        // the placeholder text is returned for XCUIElement.value
+        XCTAssertEqual(try XCTUnwrap(nameField.value as? String), "Your Name")
+        XCTAssertEqual(try XCTUnwrap(emailField.value as? String), "your.email@example.org")
+        
+        // the UITextView doesn't hav a placeholder, it's a label on top of it. so it is actually empty
+        XCTAssertEqual(try XCTUnwrap(messageTextView.value as? String), "")
+    }
+    
+    func testSubmitWithNoFieldsFilled() throws {
+        throw XCTSkip("Needs error state implementation")
+        
+        let widgetButton: XCUIElement = app.staticTexts["Report a Bug"]
+        widgetButton.tap()
+        
+        app.staticTexts["Send Bug Report"].tap()
+        
+        // TODO: need to implement some kind of error dialog and then assert for it
+    }
```

Comment:
`l`: I would prefer not to have tests that don't run if possible. They are easily overlooked, start to rot and are never updated again.

Reply:
It's implemented in https://github.com/getsentry/sentry-cocoa/pull/4544